### DISCUSSION TITLE
Help with .dll Build for android

### DISCUSSION THREAD

**Discussion by HariomJangra** [ggml-org/llama.cpp#10931] [General]
I am trying to build an react native or flutter app with the use of llama.cpp to run llm models locally on mobile. But I am not able to generate the .dll file. I have tried building from documentation but not getting the .dll file. Can anyone help me with this?

Thanks in Advance

**Comment by FSSRepo** [comment]
You need to use Android NDK and CMake

**Comment by mobiledevenv** [comment]
Android library does not use dll but so because basically Android is Linux


I found an example of Android that can be used and there is a precompiled library that can probably be implemented in your project.


https://github.com/general-developer/llama_library

**Comment by mobiledevenv** [comment]
Android library does not use dll but so because basically Android is Linux


I found an example of Android that can be used and there is a precompiled library that can probably be implemented in your project.


https://github.com/general-developer/llama_library
